### PR TITLE
UPDATED THE README FILE WITH ALL NECESSARY CHANGES

### LINKED ISSUE #680: UPDATION IN README.md FILE REQUIRED
Hello @ajay-dhangar 
the current readme file lacks proper structure and does not contain alot of things such as:

1. Featured in section 
2. Table of contents for easy access
3. back to top button 
4. contributors , forkers & stargazers section 

Kindly Assign me the issue under gsssoc-extd and hacktoberfest labels so my work counts under these events .
Thankyou

### PR DESCRIPTION
**Pull request by soham0028**
fixes: #680 

Hello @ajay-dhangar 
below are the screenshots of all the changes promised and suggested 
kindly review : 
image
image
image
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,83 @@
...
+<table>
+
+   <tr>
+      <th>Event Logo</th>
+      <th>Event Name</th>
+      <th>Event Description</th>
+   </tr>
+   <tr>
+      <td><img src="girlscript.jpg" width="200" height="auto" loading="lazy" alt="GSSoC 24"/></td>
+      <td>GirlScript Summer of Code 2024</td>
```
Comment (ajay-dhangar) [inline README.md:47]: replace `GirlScript Summer of Code 2024` with `GSSoC'24 Extd`

File: README.md [hunk 2]
```diff
@@ -1,7 +1,83 @@
...
+      <td>GirlScript Summer of Code is a three-month-long Open Source Program conducted every summer by GirlScript Foundation. It is an initiative to bring more beginners to Open-Source Software Development.</td>
```
Comment (ajay-dhangar) [inline README.md]: replace
```
GirlScript Summer of Code is a three-month-long Open Source Program conducted every summer by GirlScript Foundation. It is an initiative to bring more beginners to Open-Source Software Development.
```
with
```
GSSoC'24 Extd is a one-month open-source Program conducted by the GirlScript Foundation. It is an initiative to introduce more beginners to Open-Source Software Development.
```

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
update few code based on suggestions

**Comment by soham0028** [comment]
changes made to gssoc-extd text

**Comment by soham0028** [comment]
can you guide me with the conflicts if there are any

**Review by ajay-dhangar — APPROVED** [review]
Done

**Comment by soham0028** [comment]
thankyou

**Pull request merged**